src/App.js

```diff
@@ -13,109 +15,140 @@ import Controls from './Controls.js';
 class App extends Component {
   constructor() {
     super();
-      this.state = {
-        landingScreen: true,
-        parkData: {
-          nationalParks: [],
-        },
-        trailData: {
-          trails: [],
-        },
-        foundTrails: [],
-        trailsByPark: [],
-        selectedLocation: ''
-      }
-
+    this.state = {
+      landingScreen: true,
+      parkData: [],
+      trailData: [],
+      selectedLocation: ''
     }
+  }
 
   componentDidMount() {
-   fetch('https://whateverly-datasets.herokuapp.com/api/v1/nationalParks')
-    .then(response => response.json())
-    .then(parkData => {
-      this.setState( {
-        parkData: parkData
+    fetch('https://whateverly-datasets.herokuapp.com/api/v1/nationalParks')
+      .then(response => response.json())
+      .then(parkData => {
+        this.setState({
+          parkData: parkData.nationalParks
+        })
       })
-    })
-    .catch(error => console.log(error))  
- 
-  fetch('https://whateverly-datasets.herokuapp.com/api/v1/trails')
-    .then(response => response.json())
-    .then(trailData => {
-      this.setState( {
-        trailData: trailData
+      .catch(error => console.log(error))
+
+    fetch('https://whateverly-datasets.herokuapp.com/api/v1/trails')
```

Comment:
Is this a leftover `fetch` for trails that you meant to remove after the creation of the `fetchTrails` method below it?